ProcessMaker/Http/Controllers/Api/ProcessController.php

```diff
@@ -157,7 +154,7 @@ public function store(Request $request)
         $request->validate(Process::rules());
         $data = $request->json()->all();
 
-        if (! isset($data['status'])) {
+        if (!isset($data['status'])) {
```

Comment:
Restructer this variable to be set by default and only changed if there is a reason to.

Reply:
The default value of field status is moved to the migration process.